org.kde.angelfish.json

```diff
@@ -0,0 +1,99 @@
+{
+    "id": "org.kde.angelfish",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "base": "io.qt.qtwebengine.BaseApp",
+    "base-version": "5.15",
+    "command": "angelfish",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=pulseaudio",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--filesystem=xdg-data",
```

Comment:
What is it needed for?

Reply:
Angelfish can create desktop files for websites, and place them in ~/.local/share/applications.
However I think this doesn't fully work in flatpak because it currently detects the wrong path. I can remove it and add it back later when that's fixed if you want.

Reply:
Yeah, that would be preferred, and also limited to applications directory in the future.

Reply:
I fixed the path detection and limited the access to the applications directory now.